dask-gateway/Dockerfile

```diff
@@ -32,39 +39,40 @@ RUN useradd -m -U -u 1000 dask \
     && echo "aggressive_update_packages: []" >> /home/dask/.condarc \
     && find /opt/conda/ -follow -type f -name '*.a' -delete \
     && /opt/conda/bin/conda clean -afy \
-    && chown -R dask:dask /opt/conda \
-    && apt-get autoremove --purge -y wget bzip2 \
```

Comment:
By not removing wget / bzip2 the image will not grow further because adding + removing won't help if its not done in the same layer, and since it is no longer in the same layer I don't remove these binaries later.

The reason for having one more RUN statements was for readability.

Reply:
Do you have a rough idea of how much larger the image is now?